Makefile

```diff
@@ -64,13 +64,15 @@ INCLUDEOPTS=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
 	    | grep '^ /' | grep -v cc1plus | awk '{print "-I" $$1 " "}')
 PYINCLUDE=$(shell python -c \
 	  "import sysconfig;print(sysconfig.get_path('include'))")
```

Comment:
On 06/20, Tim Head wrote:
>> @@ -64,13 +64,15 @@ INCLUDEOPTS=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
>>  	    | grep '^ /' | grep -v cc1plus | awk '{print "-I" $$1 " "}')
>>  PYINCLUDE=$(shell python -c \
>>  	  "import sysconfig;print(sysconfig.get_path('include'))")
>
>Needs a `from __future__ import print_function;`

In this case we can abuse the print behavior in 2.7, printing a tuple unpacks
it first (so this works the same on 2.7 or 3)

>
>---
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/dib-lab/khmer/pull/1388/files/2c74498a380982257b839a0779fad8c1cdd33ae8#r67813772